src/components/Recitation.js

```diff
@@ -0,0 +1,165 @@
+import * as actions from "../redux/modules/wishlist";
+
+function Recitation({
+  year,
+  semester,
+  wishlistCourse,
+  recitation,
+  sortedRecitationsMeetings,
+  courseName,
+}) {
+  return (
+    <RecitationContainer>
+      {courseName !== recitation.name ? (
+        <h3 className="sectionName">{recitation.name}</h3>
+      ) : (
+        ""
+      )}
+      <h4 className="sectionNum">{recitation.code}</h4>
+      <Attributes
+        instructors={recitation.instructors}
+        building={splitLocation(recitation.location).Building}
+        units={convertUnits(recitation.minUnits, recitation.maxUnits)}
+        status={recitation.status}
+        type={recitation.type}
+        registrationNumber={recitation.registrationNumber}
+      />
+      <RecitationDescription>{recitation.notes}</RecitationDescription>
+
+      <DateSection sortedSectionMeetings={sortedRecitationsMeetings} />
+      <UtilBar>
+        <CalendarButton
+          onClick={() =>
+            wishlistCourse({
+              year,
+              semester,
+              course: recitation,
+            })
+          }
+        >
+          <CalendarTodayTwoTone
+            style={{
+              color: styleStatus(recitation.status),
+            }}
+          />
+          <span
+            style={{
+              color: styleStatus(recitation.status),
+            }}
+          >
+            {changeStatus(recitation)}
+          </span>
+        </CalendarButton>
+        <CalendarButton>
+          <AddBoxTwoTone
+            style={{
+              color: grey[700],
+            }}
+          />
+          <span
+            style={{
+              color: grey[700],
+            }}
+          >
+            Add to Wishlist
+          </span>
+        </CalendarButton>
+      </UtilBar>
+    </RecitationContainer>
+  );
+}
+
+Recitation.propTypes = {
+  year: PropTypes.number.isRequired,
+  semester: PropTypes.string.isRequired,
+  wishlistCourse: PropTypes.func.isRequired,
+  recitation: PropTypes.object.isRequired,
+  sortedRecitationsMeetings: PropTypes.array.isRequired,
+  courseName: PropTypes.string.isRequired,
+};
+
+const RecitationContainer = styled.div`
+  padding: 1.8vmin 2.8vmin;
+  background-color: var(--grey100);
+  width: 100%;
+  margin-left: 1%;
+  position: relative;
+
+  & > .sectionName {
+    font-size: 1.8rem;
+    font-family: var(--condensedFont);
+    color: var(--grey800);
+    margin-bottom: 0.25rem;
+  }
+
+  & > .sectionNum {
+    font-size: 1.6rem;
+    font-family: var(--condensedFont);
+    color: var(--grey700);
+    margin: 0 0 -1rem 1rem;
+  }
+
+  & > .attributes {
+    display: flex;
+    flex-wrap: wrap;
+  }
+`;
+
+const RecitationDescription = styled.div`
+  padding: 0 1.5rem 1.5rem 0.5rem;
+  max-width: 68%;
+  color: var(--grey700);
+`;
+
+const UtilBar = styled.div`
+  padding: 0.5rem;
+  height: 6vh;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  border-bottom: 1px solid;
```

Comment:
Move this line to RecitationContainer's styling. Also implement some way for this line to be hidden on the final recitation